src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
```

Comment:
The variable 'dayCost' should be defined as a constant outside of the function scope since it represents a constant value that doesn't change.